<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Combat results calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="programTitleLabel">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>AoE II board game combat results calculator</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="player1UnitsLabel">
        <property name="text">
         <string>Player 1's battle participant</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="player1EntityQuantity">
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="player1EntityNamesFilter"/>
          </item>
          <item>
           <widget class="QListWidget" name="player1EntityNames"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="player1MonkQuantity">
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Monks</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="player2UnitsLabel">
        <property name="text">
         <string>Player 2's battle participant</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout"/>
      </item>
      <item>
       <widget class="QLabel" name="gameOutputLabel">
        <property name="text">
         <string>Outcome of the battle between the battle participants:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="gameOutputTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="calculateResultsButton">
        <property name="text">
         <string>Calculate results</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="closeProgram"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QMenu" name="menuDocumentation">
     <property name="title">
      <string>Documentation</string>
     </property>
     <addaction name="actionUser_guide"/>
     <addaction name="actionDeveloper_guide"/>
    </widget>
    <addaction name="actionAbout"/>
    <addaction name="menuDocumentation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="closeProgram">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionUser_guide">
   <property name="text">
    <string>User guide</string>
   </property>
  </action>
  <action name="actionDeveloper_guide">
   <property name="text">
    <string>Developer guide</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
